composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "jimmiw/php-time-ago",
+    "name": "marlon-be/php-time-ago",
```

Comment:
hijacking? :)

Reply:
That wasn't supposed to happen. I removed the commit from the PR. :S

Reply:
@jimmiw Would it be possible to merge this PR (and create an new version)? I am using this library in a current project and would like to use your version instead of a fork.